cmd/wallet-web/src/components/CredentialDetails/DeleteCredentialModal.vue

```diff
@@ -4,92 +4,58 @@
  * SPDX-License-Identifier: Apache-2.0
 -->
 <template>
-  <modal>
-    <div
-      v-if="showModal"
-      class="
-        flex
-        overflow-y-auto
-        fixed
-        inset-0
-        z-50
-        justify-center
-        items-center
-        bg-neutrals-dark bg-opacity-50
-      "
-    >
-      <div class="relative mx-6 lg:mx-auto max-w-6xl bg-neutrals-white rounded-2xl modal-width">
-        <!--content-->
-        <div class="flex relative flex-col items-center px-8 pt-10 w-full">
-          <div class="flex justify-center items-center w-15 h-15 bg-primary-valencia rounded-full">
-            <svg width="32" height="32" xmlns="http://www.w3.org/2000/svg">
-              <g transform="translate(3 6)" fill="none" fill-rule="evenodd">
-                <rect stroke="#ffffff" stroke-width="2" x="1" y="1" width="24" height="19" rx="4" />
-                <ellipse fill="#ffffff" cx="8" cy="13" rx="4" ry="2" />
-                <circle fill="#ffffff" cx="8" cy="8" r="2" />
-                <path
-                  stroke="#ffffff"
-                  stroke-width="2"
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                  d="M16 8h5M16 13h2"
-                />
-              </g>
-            </svg>
-          </div>
-          <span class="pt-5 pb-3 text-lg font-bold text-neutrals-dark">
-            {{ t('CredentialDetails.deleteCredential') }}?
-          </span>
-          <div class="relative flex-auto">
-            <p class="pb-12 text-base text-center text-neutrals-medium">
-              {{ t('CredentialDetails.deleteCredentialConfirmMessage') }}
-            </p>
-          </div>
+  <modal :show="showModal">
+    <template #content>
+      <!--content-->
```

Comment:
Can we call this `<!-- Modal Content -->` and move to the line above the `template`?